webpack/assets/javascripts/react_app/components/HostDetails/index.js

```diff
@@ -143,10 +144,42 @@ const HostDetails = ({
                         <HostGlobalStatus hostName={id} />
                       </SplitItem>
                       <SplitItem>
-                        <Badge> {response?.operatingsystem_name}</Badge>
+                        <Label
+                          isCompact
+                          color="blue"
+                          render={({ className, content, componentRef }) => (
+                            <Link
+                              to={`/hosts?search=os_title="${response?.operatingsystem_name}"`}
+                              className={className}
+                              innerRef={componentRef}
+                              target="_blank"
```

Comment:
Why open in a new tab? The old UI's links don't do that.

Reply:
I think users don't like to be redirected somewhere, as we said for other places in the host details page(like the parameters preview) - we dont want to hurt their "flow" of being in the host details page